src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -9,6 +9,8 @@ partial class BoundStatement : IStatement
     {
         OperationKind IOperation.Kind => this.StatementKind;
 
+        bool IOperation.HasErrors => this.HasErrors;
```

Comment:
Personally, I like seeing the entire interface implementation, but of course it's not necessary. Except for giving a method inliner some exercise, it doesn't cost anything either.